inspirehep/modules/authors/forms.py

```diff
@@ -274,13 +274,26 @@ class AuthorUpdateForm(INSPIREForm):
 
     """Author update form."""
 
-    # Hidden field to hold author id information
     bai = fields.StringField(
-        widget=HiddenInput()
```

Comment:
I see you are doing a similar thing here https://github.com/inspirehep/inspire-next/pull/1550/files#diff-c8175080bad0020db2b4eea02db63e7cR201 if it is equivalent we could delete the other code.